src/main/io/ledstrip.c

```diff
@@ -179,7 +178,8 @@ void pgResetFn_ledStripConfig(ledStripConfig_t *ledStripConfig)
 }
 
 static int scaledThrottle;
-static int scaledAux;
+//static int scaledAux;
```

Comment:
Instead of commenting out code that's not used any more, remove it. If we need to get it back, it is in the git history.